### DISCUSSION TITLE
Accesing page title

### DISCUSSION THREAD

**Discussion by aligzl** [nuxt/nuxt#27171] [Questions]
Hi,
How we can access to page title which defined by useHead or useHeadSafe?
Thanks in advance

**Comment by TheAlexLichter** [comment]
Plain, non-reactive and client-side: `document.title`

Otherwise, you could [use a mutation observer](https://github.com/vueuse/vueuse/blob/db38d4c88ac073cb126fc105d9e5c8afce748a9b/packages/core/useTitle/index.ts#L93-L100).
